### PR TITLE
Add the "different threads" check to `init`

### PR DESCRIPTION
**Pull request by gkoz**
Move `gdk::set_initialized` call to `set_initialized`.

### CODE REVIEW

File: src/rt.rs [hunk 1]
```diff
@@ -93,10 +94,14 @@ pub unsafe fn set_initialized() {
 /// initialized otherwise an Err is returned.
 pub fn init() -> Result<(), ()> {
     skip_assert_initialized!();
+    if is_initialized_main_thread() {
+        return Ok(());
+    }
+    else if is_initialized() {
```
Comment (GuillaumeGomez) [inline src/rt.rs]: Can you move the `else if` one line up please?

Comment (gkoz) [inline src/rt.rs]: I'd have to fix the style in the whole file then.

Comment (GuillaumeGomez) [inline src/rt.rs]: Arf, didn't pay attention to the rest of the file. Motivated to do it?

Comment (GuillaumeGomez) [inline src/rt.rs]: I never expected it to go anywhere, that's way beyond my expectations!

**Comment by GuillaumeGomez** [comment]
Well, we'll go back on code formatting later. Thanks @gkoz!

**Pull request merged**